phpstan-baseline.neon

```diff
@@ -0,0 +1,3 @@
+parameters:
```

Comment:
I would remove this file, as we can add the `ignoreErrors` in the main neon file. Also the amount of ignore's should be limited so it should no clutter the phpstan.neon that much

Reply:
@acrobat I guess you are not aware of the baseline file mechanics.
The file can automatically be generated by phpstan therefore is a superior approach compared to manually listing ignore rules in the `phpstan.neon.dist`

See https://phpstan.org/blog/phpstans-baseline-feature-lets-you-hold-new-code-to-a-higher-standard

Reply:
I did not know this was a feature of phpstan, but would could use it in the future if needed. But for now we can leave this file away as the baseline file is empty for level 0. But thanks for the heads up, something new to learn every day 😄